OsmAnd/src/net/osmand/plus/mapcontextmenu/controllers/AmenityMenuController.java

```diff
@@ -160,12 +159,8 @@ public boolean displayDistanceDirection() {
 	public String getNameStr() {
 		String preferredLang = getPreferredMapLang();
 		if (amenity.getType().isWiki()) {
```

Comment:
this line should be inside 
 OsmandPlugin.getMapObjectPreferredLang(MapObject object, String defaultLanguage)